packages/nimble-components/src/select/styles.ts

```diff
@@ -116,6 +118,18 @@ export const styles = css`
         overflow: auto;
     }
 
+    ::slotted([role='option'][active-option]) {
+        background-color: ${fillSelectedColor};
+    }
+
+    ::slotted([role='option']:not([active-option])) {
+        background: none;
+    }
+
+    ::slotted([role='option'][active-option]:hover) {
+        background-color: ${fillHoverSelectedColor};
+    }
```

Comment:
Can these styles be owned by the list option element itself? None of it seems coupled to the state of this element

Reply:
The problem is with the middle style in particular, where we remove the background when there is no `active-option` attribute present. Currently, this will only work for the `Select`, as all other dropdowns that use the `ListOption` do not apply the `active-option` attribute, and would thus _always_ set the background to `none`, which is not what we want...yet.

Would you like me to create an issue that tracks the need to move these styles to the `ListOption` CSS?

Reply:
Either make an explicit one or capture as an item in the align list box implementations tech debt: https://github.com/ni/nimble/issues/1926

Reply:
Updated the existing tech debt issue.